### DISCUSSION TITLE
Magic Home Free GPIO Pin

### DISCUSSION THREAD

**Discussion by boxxeronfly** [arendst/Tasmota#10929] [Show and tell]
I am not sure if this the correct place to put this information. But I wanted to save someone the effort to figure this out.

I was able to use GPIO15 (D8) by soldering to an exposed pad.  I added a PIR sensor to it.
![Magic_Home_Top_Side_GPIO15](https://user-images.githubusercontent.com/1420652/107719946-43962000-6c8e-11eb-8a04-479099c4edd9.jpg)

**Comment by barbudor** [comment]
Thank you for pointing this but this is true only if the 4th channel is not installed.
I have a 4 channel MagicHome RGBW where this pin is not available.

However the pin GPIO4 used for the IR remote control receiver can also be used on the pad labelled "IR" but includes a 20K pull down (the "203") and a 100R serie resistor (the "101")
With 3V3 and GND close to it, that is also a very convenient place for a PIR, a DS18x20, DHT, ...

![image](https://user-images.githubusercontent.com/2996042/107741429-6e738900-6d0d-11eb-8640-b735286b8417.png)
